src/dynamo-streams.ts

```diff
@@ -258,7 +258,6 @@ export class DynamoStreamHandler<Entity, Context> {
             );
           },
           concurrency: this.config.concurrency ?? 5,
-          stopOnError: false,
```

Comment:
The change in `processWithOrdering` keeps the same behavior as before (when it always had `stopOnError` set to `false`. Now, `processWithOrdering` keeps track of errors that it has encountered, and if `rejectOnError` is set to `true`, which it is by default, it will explicitly throw an `AggregateError`. See https://github.com/lifeomic/delta/pull/59/files#diff-39b2554fd18da165b59a6351b1aafff3714e2a80c1435f2de9706355b4d32351R108.